docs/classes-overview.md

```diff
@@ -0,0 +1,19 @@
+## Classes diagram
+
+This diagram is intended to help understand how the library works internally. It may leak some implementation details and should not be used as API.
```

Comment:
@wyrzyk This is relly well-written description to well-written diagram. 
Helps dive into the project.
I'm sure it'll pay off long term to make a decision, use common languages, and helps also other ppl contribute and maintain the library.
㊗️

Reply:
㊗️ = congratulations? 😃

Reply:
![Screenshot 2021-02-19 at 12 24 58](https://user-images.githubusercontent.com/77149850/108498671-7dc67980-72ad-11eb-946f-72a1f9c266dd.png)
jej, this is what github prompt me;P